__tests__/ship.test.js

```diff
@@ -4,20 +4,133 @@ describe("constructor", () => {
   it("returns an object", () => {
     expect(new Ship("Enterprise")).toBeInstanceOf(Object);
   });
-
+  const ship = new Ship("Enterprise");
   it("sets the name property", () => {
-    const ship = new Ship("Enterprise");
-
     expect(ship.name).toEqual("Enterprise");
   });
+  it("initial year of 0", () => {
+    expect(ship.yearsInService).toEqual(0);
+  });
+  it("initial fuel of 10", () => {
+    expect(ship.fuel).toEqual(20);
+  });
+  it("initial morale of 8", () => {
+    expect(ship.morale).toEqual(8);
+  });
```

Comment:
Bit of formatting here, its good practice to leave a line clear between each test so to make it more readable. Also a typo in the name of the 'initial fuel' test:
```
  it("initial year of 0", () => {
    expect(ship.yearsInService).toEqual(0);
  });

  it("initial fuel of 20", () => {
    expect(ship.fuel).toEqual(20);
  });

  it("initial morale of 8", () => {
    expect(ship.morale).toEqual(8);
  });
```